libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -219,9 +274,19 @@ class AC_AutoTune {
     AP_Float min_d;
 
     // copies of object pointers to make code a bit clearer
-    AC_AttitudeControl_Multi *attitude_control;
+    AC_AttitudeControl *attitude_control;
     AC_PosControl *pos_control;
     AP_AHRS_View *ahrs_view;
     AP_InertialNav *inertial_nav;
     AP_Motors *motors;
+
+    uint8_t  tune_seq[6];                // holds sequence of tune_types to be performed
```

Comment:
maybe this should be an array of TuneType?

Reply:
done.  Thanks!